docs/src/user-guide/command-line-interface.md

```diff
@@ -142,9 +149,14 @@ If `.eslintrc.*` and/or `package.json` files are also used for configuration (i.
 
 #### `--env`
 
-This option enables specific environments. Details about the global variables defined by each environment are available on the [Specifying Environments](configuring/language-options#specifying-environments) documentation. This option only enables environments; it does not disable environments set in other configuration files. To specify multiple environments, separate them using commas, or use the option multiple times.
+This option enables specific environments.
 
-Examples:
+* **Argument Type**: String. One of the available environments.
+* **Multiple Arguments**: Yes
+
+Details about the global variables defined by each environment are available in the [Specifying Environments](configuring/language-options#specifying-environments) documentation. This option only enables environments. It does not disable environments set in other configuration files. To specify multiple environments, separate them using commas, or use the option multiple times.
```

Comment:
Just double checking -- specifying vs. specify in the link?

Reply:
yep, this is currently the name of the section